KeyboardTeacher/Box.pde

```diff
@@ -12,6 +12,8 @@ class Box {
     selfHeight = H;
     text = T;
     textSize = selfWidth / text.length();
+    textAlign(CENTER, CENTER);

```

Comment:
Be aware that this setup instruction may be overwritten before the relative drawing function. That is while the local variables (in this case, textFont) live inside the objects, and no one can change them, the textAlign function modifies global variables that anyone else can change. Is it clear?

In other words, you should put all the formatting functions just before the relative rendering functions. In this case, the rendering function is `text()`, so you should restore the textAlign function shortly before it.